pwndbg/commands/context.py

```diff
@@ -162,11 +163,26 @@ def context_disasm():
 
     return banner + result
 
+# XXX: move this part to color/ just like color/disasm.py:instrution()?
 theme.Parameter('highlight-source', True, 'whether to highlight the closest source line')
-
+source_code_length = pwndbg.config.Parameter('context-source-code-length',
+                                             10,
+                                             'number of lines of source code to printed by the context command')
+@pwndbg.memoize.reset_on_exit
```

Comment:
heh I wonder what's the difference between `reset_on_exit` and `reset_on_start`.

We probably should cache it untill next run of the same binary (as someone might have changed the code and recompile it).

Reply:
I read the code but still don't know when will trigger `start` and `exit` event. I think they are similar in this case. I suggest that someone know the memorize and event can write some doc/comments for each event about when gdb will file it and use case.

BTW, I change to `reset_on_start`. It looks more consistent for me.